sdks/python/apache_beam/runners/portability/fn_api_runner/fn_runner_test.py

```diff
@@ -2002,6 +2019,26 @@ def test_gbk_many_values(self):
       assert_that(r, equal_to([VALUES_PER_ELEMENT * NUM_OF_ELEMENTS]))
 
 
+# TODO(robertwb): Why does pickling break when this is inlined?
+class CustomMergingWindowFn(window.WindowFn):
+  def assign(self, assign_context):
+    return [
+        window.IntervalWindow(
+            assign_context.timestamp, assign_context.timestamp + 1)
```

Comment:
Could we increase `assign_context.timestamp + 1` to `maybe assign_context.timestamp + 100`? It seems that for dataflow runner v2, in batch  the windows are inserted and fired in order, window like [2, 3) could already been fired when element with timestamp 100 arrives, and the merging behavior may be inconsistent with direct runner or ULR.

Reply:
Good point. Done. PTAL.